site/docs/spec.md

```diff
@@ -416,25 +530,91 @@ Notes:
 
 ### Delete Formats
 
-This section details how to encode row-level deletes in Iceberg metadata. Row-level deletes are not supported in the current format version 1. This part of the spec is not yet complete and will be completed as format version 2.
+This section details how to encode row-level deletes in Iceberg delete files. Row-level deletes are not supported in v1.
+
+Row-level delete files are valid Iceberg data files: files must use valid Iceberg formats, schemas, and column projection. It is recommended that delete files are written using the table's default file format.
```

Comment:
Most organizations prefer one format to another, so we just want to set the expectation that if a table is written with ORC, deletes should also be written with ORC. That way if there is a problem with the other format, it only affects the people that use it.

For example, we don't use ORC so there may be a problem actually loading and using it in our environment. We don't really exercise it in our integration tests. So I wouldn't want applications built with the assumption that they should be able to write deltas as ORC files all the time because it is allowed by the spec.

Reply:
If I remember correctly, Hive also allows configuring differential files separately from base files (properties like target file size, row-group size, etc). I think that will also apply in our case (we can figure out details while implementing minor and major compactions).